addons/web_editor/controllers/main.py

```diff
@@ -551,11 +551,11 @@ def shape(self, module, filename, **kwargs):
                 user_colors.append([tools.html_escape(value), colorMatch.group(1)])
```

Comment:
Commit message: 
- You should add steps to reproduce the issue (like in the task) :wink: It's always a good thing to have steps when doing a bugfix :wink:  Something like this 

  Steps to reproduce:
  - On Firefox, drop the Image-Text snippet
  - ... 

- Maybe add a brief explanation on why it bugs only on Firefox (as you explained in the task, because firefox parses the svg differently ?) 

- > transformation is applied to each of them, causing unexpected
behaviors.

  A detail but there is enough space on the first line to put "behaviors." on the same line. Just telling in case you thought like me that 72 char = the 72nd column :stuck_out_tongue: